### PR TITLE
feat: support skipDuplicatesCheck in AddPrimaryKeyHandler

### PR DESCRIPTION
**Pull request by vojtabiberle**
Wrap the duplicate detection query in a conditional so it is skipped when AddPrimaryKeyCommand.skipDuplicatesCheck is true. Add functional test testDuplicatesSkipCheck to verify PK creation succeeds with duplicates when the flag is set.

Linear: [DMD-981](https://linear.app/keboola/issue/DMD-981/implement-skipduplicatescheck-for-bigquery-driver)
Connection PR: https://github.com/keboola/connection/pull/6678

---

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
         "google/protobuf": "^3.21",
         "keboola/php-file-storage-utils": "^0.2.5",
         "ext-json": "*",
-        "keboola/storage-driver-common": "^7.14.0",
+        "keboola/storage-driver-common": "dev-vb/DMD-981/bq_skipDuplicatesCheck#b08f8b7127df06a199e2d88c032171189c4a5c40 as 7.15.0",
```
Comment (vojtabiberle) [inline composer.json]: use proper version

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -17,9 +17,13 @@ jobs:
...
         run: docker compose build production
 
+      # TODO: revert back to `composer ci` once storage-driver-common is released
       - name: Check
         run: |
-          docker compose run --rm production composer ci
+          docker compose run --rm production composer phplint
+          docker compose run --rm production composer phpcs
+          docker compose run --rm production composer phpstan
+          docker compose run --rm production composer tests-unit
```
Comment (vojtabiberle) [inline .github/workflows/tests.yml]: revert back to `docker compose run --rm production composer ci`

**Review by zajca — COMMENTED** [review]
Cajk jak bude vydaný driver tak máš 👍

**Review by zajca — APPROVED** [review]
